app/views/sections/items/ItemQuantityView.scala.html

```diff
@@ -39,37 +39,42 @@
 
 @(form: Form[_], action: Call, goodsType: GoodsType, cnCodeInfo: CnCodeInformation, idx: Index)(implicit request: DataRequest[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("itemQuantity.title", goodsType.toSingularOutput())), maybeShowActiveTrader = maybeShowActiveTrader(request)) {
+@layout(pageTitle = title(form, messages(itemQuantityHelper.title(goodsType, cnCodeInfo))), maybeShowActiveTrader = maybeShowActiveTrader(request)) {
+    @formHelper(action) {
 
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        } else {
+            @errorNotificationBannerHelper.content(Seq(ItemQuantityPage(idx).getSubmissionErrorCode(isOnAddToList = false)).flatten, hasContentHeading = false).map(notificationBanner(_))
+        }
 
-  @formHelper(action) {
+        @h2(messages("items.subHeading", request.draftId), "govuk-caption-xl", hiddenContent = Some(messages("subHeading.hidden")))
 
-    @if(form.errors.nonEmpty) {
-      @govukErrorSummary(ErrorSummaryViewModel(form))
-    } else {
-      @errorNotificationBannerHelper.content(Seq(ItemQuantityPage(idx).getSubmissionErrorCode(isOnAddToList = false)).flatten, hasContentHeading = false).map(notificationBanner(_))
-    }
-
-    @h2(messages("items.subHeading", request.draftId), "govuk-caption-xl", hiddenContent = Some(messages("subHeading.hidden")))
+        @if(itemQuantityHelper.requiresRicherContent(cnCodeInfo)) {
+            @itemQuantityHelper.heading(cnCodeInfo).map { heading => @h1(heading) }
+            @itemQuantityHelper.paragraph(cnCodeInfo).map{ paragraph => @p(){@paragraph} }
+            @inputField(LabelSize.Medium)
+        } else {
+            @inputField(LabelSize.Large)
+        }
 
-    @h1(itemQuantityHelper.heading(goodsType, cnCodeInfo))
-
-    @itemQuantityHelper.paragraph(cnCodeInfo).map{ paragraph => @p(){@paragraph} }
+        @continueOrExit()
+    }
+}
 
+@inputField(labelSize: LabelSize) = {
     @govukInput(
         InputViewModel(
             field = form("value"),
-            label = LabelViewModel(Text(itemQuantityHelper.label(cnCodeInfo))).asPageHeading(LabelSize.Medium)
+            label = LabelViewModel(Text(itemQuantityHelper.label(goodsType, cnCodeInfo))).asPageHeading(labelSize)
```

Comment:
This can only do `.asPageHeading` if it's the scenario where it's the H1 of the page (when there's no richer content). 

Otherwise, on the pages where it does have richer content, you end up with two `H1` tags which would also get flagged by accessibility:

![Screenshot 2024-08-16 at 14 20 20](https://github.com/user-attachments/assets/1e47a0f8-3be2-4ada-b4a7-0c8b3ea9eb43)

Reply:
Fixed
<img width="1129" alt="image" src="https://github.com/user-attachments/assets/9c8dd15d-26e8-4ee5-89d2-b33f2bfbe90d">